bin/tv-train

```diff
@@ -8,15 +8,17 @@ from __future__ import division
 from __future__ import print_function
 
 import imp
+import pdb
```

Comment:
Why do you import this? (Seems not to be necessary for the code; only for debugging)